Restore ability to run ginkgo on OSX

tgraf opened the pull request:
This restores the ability to run ginkgo and the integration tests on OSX by removing Linux specific go packages from the path.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cilium/cilium/4443)
<!-- Reviewable:end -->

manalibhutiyani commented on test/runtime/chaos.go:
```diff
@@ -71,7 +70,7 @@ var _ = Describe("RuntimeValidatedChaos", func() {
 		//
 		// jq 'map(select(.status.identity.id != 4), del(.status.controllers, ..., (.status.identity.labels | sort)))'
-		filterHealthEP := fmt.Sprintf("select(.status.identity.id != %d)", identity.GetReservedID("health"))
+		filterHealthEP := "select(.status.identity.id != 4)" // ReservedIdentityHealth=4
```
Can we declare a const, or use an existing variable instead of hard-coding 4

tgraf replied:
I'm fine with a const as long as we don't use the one in the API definition. I'll add it.

manalibhutiyani commented on pkg/byteorder/byteorder.go:
```diff
@@ -6,7 +6,7 @@
 //     http://www.apache.org/licenses/LICENSE-2.0
 //
-// Unless required by applicable law or agreed to in writing, software
+// Usless required by applicable law or agreed to in writing, software
```
Nit: Change this back to the original correct `Unless` instead of `Usless`

tgraf replied:
Ooops ;-)

manalibhutiyani reviewed (approved):
looks good. Pointed out some nits

tgraf committed:
test: Hardcode identity for health endpoint

Improves the test by not depending on code that can be changed for a stable API
and removes dependency of identity package.

Signed-off-by: Thomas Graf <thomas@cilium.io>

tgraf committed:
byteorder: Do not depend on external Linux only library

Signed-off-by: Thomas Graf <thomas@cilium.io>
